app/models/user_challenge.rb

```diff
@@ -1,2 +1,6 @@
 class UserChallenge < ApplicationRecord
-end
+    validates :completed, presence: true
+    validates :notes, presence: true, length: { maximum: 1500, }
```

Comment:
I think a user challenge should be able to be created with out notes or feedback and both can be added at a later date. so we shouldn't validate presence true for those two fields